packages/react-form-state/docs/FAQ.md

```diff
@@ -38,6 +38,53 @@ class MyComponent extends React.Component {
 
 If you need to do something immediately on mount you could also use old fashioned [callback refs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs).
 
+## My form keeps resetting for no reason! / My form is resetting whenever I change an input!
```

Comment:
this is probably the most asked question I get, so I snuck it in here since it's sorta on theme